src/main/kotlin/fi/metatavu/timebank/api/controllers/UsersController.kt

```diff
@@ -0,0 +1,36 @@
+package fi.metatavu.timebank.api.controllers
+
+import fi.metatavu.timebank.api.impl.translate.UserTranslator
+import fi.metatavu.timebank.api.keycloak.KeycloakController
+import fi.metatavu.timebank.model.User
+import java.util.UUID
+import javax.enterprise.context.ApplicationScoped
+import javax.inject.Inject
+
+@ApplicationScoped
```

Comment:
This project does use class and method level documentation. Please include them, even though both the class and its methods are rather self documenting.